website/templates/project/project_base.mako

```diff
@@ -1,5 +1,25 @@
 <%inherit file="../base.mako"/>
 
+<%def name="resource()"><%
+        if node.get('is_registration', False):
+           return 'registration'
+        elif node.get('is_preprint', False):
+           return 'preprint'
```

Comment:
Is this going to be confusing?  So the node that is associated with a preprint is being accessed here, but not the preprint itself.  For the actual preprint to send to GA that it is a preprint, that change would need to be made ember-side.

Reply:
I've talked to Project about this. They are okay with this for now, actual preprints that are to be covered by the Interfaces (I assume) and will will require some coordination at a later date.